socorro/tests/stage_submitter/test_submitter.py

```diff
@@ -0,0 +1,594 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
The tests were lifted from socorro-submitter and then adjusted to work with the SubmitterApp instead of the socorro-submitter lambda function.